google-http-client/src/test/java/com/google/api/client/http/HttpResponseTest.java

```diff
@@ -461,6 +462,21 @@ public LowLevelHttpResponse execute() throws IOException {
   }
 
   public void testGetContent_gzipEncoding_finishReading() throws IOException {
+    do_testGetContent_gzipEncoding_finishReading("gzip");
+  }
+
+  public void testGetContent_gzipEncoding_finishReadingWithUppercaseContentEncoding() throws IOException {
```

Comment:
needs `@Test`; writing the test first and making sure it fails helps avoid problems like this

Reply:
This is an existing old test class using JUnit 3 method conventions thus the lack of the @Test annotation. It runs fine as-is.